experiment/simulation/js/flipflop.js

```diff
@@ -0,0 +1,497 @@
+import { registerGate,jsPlumbInstance } from "./main.js";
+import { setPosition } from "./layout.js";
+import { gates,printErrors } from './gate.js';
+
+'use strict';
+
+export let flipFlops = {};
+
+export function clearFlipFlops() {
+
+    for (let ffID in flipFlops) {

```

Comment:
Do we need this block? Generally it's not a good idea to delete from a collection while iterating over it

Reply:
Just assigned the object to {}